metaci/notification/tasks.py

```diff
@@ -36,10 +37,13 @@ def queue_build_notifications(build_id):
     repo_notifications = RepositoryNotification.objects.filter(repo = build.repo, **status_query).values('user__id').distinct()
     branch_notifications = BranchNotification.objects.filter(branch = build.branch, **status_query).values('user__id').distinct()
     plan_notifications = PlanNotification.objects.filter(plan = build.plan, **status_query).values('user__id').distinct()
+    planrepository = PlanRepository.objects.filter(plan=build.plan, repo=build.repo)[0]
```

Comment:
can we use `PlanRepository.objects.get` instead of the 0 index?

Reply:
No because it may return more than 1 object. Although any duplicates would have the exact same info, we currently don't have protection against duplicates existing in the first place.

Reply:
:( fair enough. we should definitely have a unique index there, but not in scope this PR.